jdaviz/configs/default/plugins/collapse/collapse.py

```diff
@@ -16,28 +14,15 @@
 __all__ = ['Collapse']
 
 
-spaxel = u.def_unit('spaxel', 1 * u.Unit(""))
```

Comment:
This is not used. Not sure why it is here. It is also defined in at least two other places in this package but removing those is out of scope here.